app-web/src/components/Search/index.js

```diff
@@ -45,12 +45,15 @@ export const TEST_IDS = {
 };
 
 export const Search = ({ onSearch, searchOnEnter, inputConfig, query, ...rest }) => {
-  const [terms, setTerms] = useState('');
-
+  const [terms, setTerms] = useState(query);
```

Comment:
im not sure if we need to preset this state with query

Reply:
I put query here because I want query still stay in the input box after refresh current page